query_optimizer/logical/CreateTable.hpp

```diff
@@ -100,8 +108,9 @@ class CreateTable : public Logical {
   static CreateTablePtr Create(
       const std::string &relation_name,
       const std::vector<expressions::AttributeReferencePtr> &attributes,
-      const std::shared_ptr<const StorageBlockLayoutDescription> &block_properties) {
-    return CreateTablePtr(new CreateTable(relation_name, attributes, block_properties));
+      const std::shared_ptr<const StorageBlockLayoutDescription> &block_properties,
+      const std::shared_ptr<const serialization::PartitionSchemeHeader> &partition_scheme_header_proto) {
```

Comment:
Add a comment for `partition_scheme_header_proto` in method header (and mention that `partition_scheme_header_proto` can be `nullptr` if not specified).